demo/site/src/documents/pages/blocks/KeyFactItemBlock.tsx

```diff
@@ -3,44 +3,31 @@ import { type KeyFactsItemBlockData } from "@src/blocks.generated";
 import { defaultRichTextInlineStyleMap, RichTextBlock } from "@src/common/blocks/RichTextBlock";
 import { Typography } from "@src/common/components/Typography";
 import { type Renderers } from "redraft";
-import styled from "styled-components";
+
+import styles from "./KeyFactItemBlock.module.scss";
 
 const descriptionRenderers: Renderers = {
     inline: defaultRichTextInlineStyleMap,
 };
 
 export const KeyFactItemBlock = withPreview(
     ({ data: { icon, fact, label, description } }: PropsWithData<KeyFactsItemBlockData>) => (
-        <Root>
-            {icon.damFile && <Icon data={icon} width={48} height={48} />}
-            <FactTypography variant="h500">{fact}</FactTypography>
+        <div className={styles.root}>
+            {icon.damFile && (
+                <div className={styles.icon}>
+                    <SvgImageBlock data={icon} width={48} height={48} />
+                </div>
```

Comment:
The SvgImageBlock accepts className, can we remove the extra div?

Reply:
Strangely enough it worked with styled components

Reply:
Please merge main and remove the div, thanks!